src/events/multi-asset.upsert.service.ts

```diff
@@ -35,12 +35,21 @@ export class MultiAssetUpsertService {
         operation.type === BlockOperation.CONNECTED ||
         operation.type === BlockOperation.DISCONNECTED;
 
-      if (shouldUpsertMultiAsset) {
+      if (
+        shouldUpsertMultiAsset &&
+        this.occurredAfterLaunch(operation.block.timestamp)
+      ) {
         await this.upsert(operation);
       }
     }
   }
 
+  occurredAfterLaunch = (comparisonDate: Date): boolean => {
```

Comment:
nit for consistency with the rest of the code
```suggestion
  private occurredAfterLaunch(comparisonDate: Date): boolean {
```